packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -716,7 +716,15 @@ class InspectorReferenceData {
 }
 
 // Production implementation of [WidgetInspectorService].
-class _WidgetInspectorService = Object with WidgetInspectorService;
+class _WidgetInspectorService with WidgetInspectorService {
+  _WidgetInspectorService() {
+    selection.addListener(() {
+      if (selectionChangedCallback != null) {
```

Comment:
nit: use pattern
`final selectionChangedCallback = this.selectionChangedCallback;`
to avoid dupe null check due to limitations of Dart type inference.